limitador-server/src/metrics.rs

```diff
@@ -0,0 +1,291 @@
+use std::collections::HashMap;
+use std::ops;
+use std::time::{Duration, Instant};
+use tracing::span::{Attributes, Id};
+use tracing::Subscriber;
+use tracing_subscriber::layer::{Context, Layer};
+use tracing_subscriber::registry::LookupSpan;
+
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub struct Timings {
+    idle: u64,
+    busy: u64,
+    last: Instant,
+}
+
+impl Timings {
+    fn new() -> Self {
+        Self {
+            idle: 0,
+            busy: 0,
+            last: Instant::now(),
+        }
+    }
+}
+
+impl ops::Add for Timings {
+    type Output = Self;
+
+    fn add(self, _rhs: Self) -> Self {
```

Comment:
It is "unusual" to prefix a variable with an underscore, when you actually use it... any reason?

Reply:
Yep good question that is strange, I just used the variable that my IDE auto-generated for me but I guess it was because I wasn't using it yet - I'll change